src/components/LightweightChart/LightweightChart.tsx

```
@@ -0,0 +1,226 @@
+import { log } from '@/lib/telemetry';
+import { mapCandle } from '@/lib/tradingView/utils';
+import { useAppSelector } from '@/state/appTypes';
+import { getAppTheme } from '@/state/appUiConfigsSelectors';
+import {
+  CandlestickData,
+  CandlestickSeries,
+  ColorType,
+  createChart,
+  IChartApi,
+  ISeriesApi,
+  Time
```

ESLint

[error] Insert `,`

(prettier/prettier)